security/webauthn/src/test/java/io/quarkus/ts/security/webauthn/AbstractWebAuthnTest.java

```diff
@@ -47,7 +55,12 @@ enum User {
     @BeforeAll
     public static void setup() {
         cookieFilter = new CookieFilter();
+    }
 
+    @BeforeEach
+    public void setupUrl() throws MalformedURLException {
+        URILike http = getApp().getURI(Protocol.HTTP);
+        url = new URL(http.getRestAssuredStyleUri() + ":" + http.getPort());
```

Comment:
Can be replaced with
```suggestion
        url = new URL(getApp().getURI(Protocol.HTTP).toString());
```
(with drop of URILike import)

Reply:
I didn't know this would work. Took me quite some time to come up with this TBH 😅